sentinelsat/scripts/cli.py

```diff
@@ -68,9 +78,7 @@ def convert(self, value, param, ctx):
     show_default=True,
     help="End date of the query in the format YYYYMMDD.",
 )
-@click.option(
-    "--geometry", "-g", type=click.Path(exists=True), help="Search area geometry as GeoJSON file."
-)
+@click.option("--geometry", "-g", type=str, help="Search area geometry as GeoJSON file.")
```

Comment:
The help string needs to be extended to reflect the change that now also GeoJSON strings and WKT strings are supported.